### PR TITLE
Add Lab. Informática e Sistemas @ IPN

### PR DESCRIPTION
**Pull request by rjmAmaro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | [Armis](http://www.armisgroup.com/) [:rocket:](http://www.armisgroup.com/ofertas-de-trabalho/) | Technology development and consultancy. | `Lisboa` `Porto` |
 | [Critical Software](https://www.criticalsoftware.com) [:rocket:](https://www.criticalsoftware.com/careers) | Mission and business-critical applications. | `Coimbra` `Lisboa`<br>`Porto` |
 | [Imaginary Cloud](https://www.imaginarycloud.com) [:rocket:](https://www.imaginarycloud.com/careers) | Web, mobile, development and design services. | `Lisboa` |
+| [Laboratório de Informática e Sistemas @ IPN](https://www.ipn.pt/laboratorio/LIS) | Technological R&D and innovative multi-disciplinary projects. | `Coimbra` |
```
Comment (marmelo) [inline README.md]: Is it possible to shorten the name to `IPN LIS`?  
Or else the table formatting will be destroyed.
Thanks!

Comment (rjmAmaro) [reply README.md]: Yes, it is. Gonna do that.

**Comment by rjmAmaro** [comment]
@marmelo changes made.

**Comment by marmelo** [comment]
Thanks @rjmAmaro! :star:

**Pull request merged**